lisa/tools/reboot.py

```diff
@@ -9,7 +9,7 @@
 from func_timeout import FunctionTimedOut, func_set_timeout  # type: ignore
 
 from lisa.executable import Tool
-from lisa.features import SerialConsole
+from lisa.features import SerialConsole, StartStop
```

Comment:
Don't use the feature in tool. If a test case need to be reboot by platform, the test case should invoke StartStop feature, instead of warping the feature into reboot.